test/membrane_mp4/demuxer/isom/demuxer_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule Membrane.MP4.Demuxer.ISOM.IntegrationTest do
+  use ExUnit.Case, async: true
+
+  import Membrane.Testing.Assertions
+
+  alias Membrane.ParentSpec
+  alias Membrane.Testing.Pipeline
+
+  # Fixtures used in demuxer tests below were generated with `chunk_duration` option set to `Membrane.Time.seconds(1)`.
+
+  defp assert_files_equal(file_a, file_b) do
+    assert {:ok, a} = File.read(file_a)
+    assert {:ok, b} = File.read(file_b)
+    assert a == b
+  end
+
+  defp ref_path_for(filename), do: "test/fixtures/payloaded_#{filename}"
+
+  defp perform_test(pid, filename, out_path) do
+    ref_path = ref_path_for(filename)
+
+    assert_end_of_stream(pid, :sink, :input)
+    refute_sink_buffer(pid, :sink, _buffer, 0)
+
+    assert :ok == Pipeline.terminate(pid, blocking?: true)
+
+    assert_files_equal(out_path, ref_path)
+  end
+
+  describe "Demuxer.ISOM should demux" do
+    @tag :skip
```

Comment:
```suggestion
```

Reply:
Well, the demuxer barely exists now, so the tests will fail.

Reply:
ok, thought it's a leftover